### ISSUE TITLE
after update to orca slicer 2.1 i cant see the printer tab in prepare so i cant change any settings for the slice

### ISSUE THREAD

**Issue by maxxx505** [SoftFever/OrcaSlicer#5824]
### Is there an existing issue for this problem?

- [X] I have searched the existing issues

### OrcaSlicer Version

2.1

### Operating System (OS)

Windows

### OS Version

windows 11

### Printer

ender 3 v3 ke

### How to reproduce

i updated the slicear and then i cant see it

### Actual results

i cant slice anything with my settings

### Expected results

usinng slicer how i want

### Project file & Debug log uploads

??

### Checklist of files to include

- [X] Log file
- [ ] Project file

**Comment by Avpman2** [comment]
I believe this happened to me, also. I installed the new version (prompted when the old program loaded), and all my printers and settings were gone when the new version ran. Reinstalling the previous version did not fix it. I had to restore my PC to the point before the update to the latest version. From now on, I will test the "portable" version so as not to muck up my installed version.

**Comment by BrunoBrutus** [comment]
I'm only using the portable version, and I'm experiencing the same issue. I had three different printer profiles, but now the system prompts for a fresh start each time. 
First I added a new printer (BAMBU P1S) and removed the other selections. 
After restarting the program, I added generic KLIPPER and MARLIN 0.4 nozzle printers, and the other printers appeared. 
Then, I removed the Bambu, and the remaining printers persisted after restarting the program.
